packages/gatsby-theme-newrelic/gatsby-node.js

```diff
@@ -320,11 +326,9 @@ exports.onCreateWebpackConfig = ({ actions, plugins }, themeOptions) => {
 };
 
 exports.onPostBootstrap = (_, themeOptions) => {
-  const { refetch, file } = themeOptions;
-
-  if (refetch) {
+  if (themeOptions.swiftype && themeOptions.swiftype.refetch) {
     fs.writeFileSync(
-      file,
+      themeOptions.swiftype.file,
```

Comment:
Now that I see this in the theme, should we add a validation in `onPreInit` to make sure this option is given if `swiftype` is defined? We'll need to make sure this file exists.

We may also want something in `onPreBootstrap` to create this file for you if its not already been created.

Reply:
I'll just add another conditional check for now, since I we need to get the fixes in this PR released soon.